### PR TITLE
Add bug class to support query advisory list from bug id 

### PR DESCRIPTION
**Pull request by shiywang**
Fixes https://github.com/red-hat-storage/errata-tool/issues/174

@sosiouxme  currently I just added a subcommand to test it out.  (https://errata.devel.redhat.com/bugs/1723928/advisories)
```
[jenkins@buildvm errata-tool]$ bin/errata-tool bug get_all_advisory_ids 1723928
43686
```

### CODE REVIEW

File: errata_tool/cli/bug.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    """
+    Get information about build
+
+    :param args: parsed cli arguments
+    """
+    build = Bug(args.id)
+    print(build)
+
+
+def get_all_advisory_ids(args):
```
Comment (shiywang) [inline errata_tool/cli/bug.py]: should we use the term `advisory` or `erratum` or `errata` ?

Comment (sosiouxme) [reply errata_tool/cli/bug.py]: As much as I wish we could do away with the terms `errata` and `erratum` and just use `advisory`, I see that in the rest of the code there are references to `errata_id` and none to `advisory_id`. So, for consistency (fewer ways to forget / be confused), I think new code should keep doing that.

I would also shorten this to the slightly less verbose
```suggestion
def get_errata_ids(args):
```

**Comment by sosiouxme** [comment]
So - seems good to me -- @ktdreyer any concerns?

**Comment by ktdreyer** [comment]
This looks good. Would you mind writing a simple unit test for this class so that we are sure we do not break it?

**Comment by shiywang** [comment]
@sosiouxme @ktdreyer done. PTAL

File: errata_tool/bug.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+from errata_tool import ErrataConnector
+
+
+class Bug(ErrataConnector):
+    def __init__(self, id):
+        """
```
Comment (yazug) [inline errata_tool/bug.py]: docstring linting prefers no leading whitespace between """ and first word of summary

**Comment by yazug** [comment]
https://github.com/red-hat-storage/errata-tool/issues/179 issue also affects this pull request.

**Comment by yazug** [comment]
I rebased this and left attribution to you in https://github.com/red-hat-storage/errata-tool/pull/201

**Comment by sosiouxme** [comment]
closing favor of PR #201
